src/transforms.jl

```diff
@@ -40,15 +40,30 @@ clrinv(x) = Composition(𝓒(exp.(x)))
 
 Isometric log-ratio transformation of composition `c`.
 """
-function ilr(c::Composition)
-  # TODO
-end
+ilr(c::Composition{D,SYMS}) where {D,SYMS} =
+  SVector{D-1}(-helmert(D) * log.(c.parts .+ eps()))
 
 """
     ilrinv(x)
 
 Inverse ilr for coordinates `x`.
 """
-function ilrinv(x)
-  # TODO
+ilrinv(x::SVector{D,T}) where {D,T<:Real} =
+  Composition(SVector{D+1}(𝓒(exp.(-transpose(x) * helmert(D+1)))))
+
+"""
+  helmert(n)
+
+Create an Helmert matrix of order `n`.
+Based on `scipy.linalg.helmert` python code
```

Comment:
I think we can leave this `scipy.linalg.helmert` comment out. It is great that the code helped in the implementation, but we don't need to add it to the docstring.

Reply:
Ok!

Reply:
Thanks! You don't need to mark as resolved manually, github will update this when you push more commits.